pom.xml

```diff
@@ -22,8 +22,8 @@
         <skipTests>true</skipTests>
         <test.excludes>**/*Test.java.bogus</test.excludes>
 
-        <kfs.version>2017-12-14</kfs.version>
-        <rice.version>2.6.1</rice.version>
+        <kfs.version>2018-02-15</kfs.version>
+        <rice.version>2.6.2</rice.version>
         <cynergy.version>2.6.1</cynergy.version>
```

Comment:
We would need to change that after the cynergy upgrade is ready, correct?

Reply:
ok

Reply:
I've pushed my current 2.6.2 changes to the KFSPTS-11127 branches of the relevant Cynergy projects, if you want to start testing the use of Cynergy 2.6.2 by building and installing the branch code locally.

Also, now that this cu-kfs PR is available, I should be able to move forward on my remaining Cynergy 2.6.2 local testing so that we can get that wrapped up as well.